<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeInformation</class>
 <widget class="QDialog" name="VolumeInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Information</string>
  </property>
  <widget class="QLabel" name="lblLabelPath">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLabelDomain">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>112</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Domain:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLabelVoxel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Voxel Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblImagePath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>31</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblImageDomain">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>112</y>
     <width>391</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblVoxelSize">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>71</y>
     <width>421</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblLabelDepth">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>150</y>
     <width>341</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblDepth">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Radiometric resolution:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxAndMinValues">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>511</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
